src/FormBuilder/resources/js/form-builder/src/settings/donation-goal/index.tsx

```diff
@@ -69,11 +69,11 @@ const DonationGoalSettings = () => {
                         />
                     </PanelRow>
                     <PanelRow>
-                        <NumberControl
+                        <CurrencyControl
```

Comment:
@JasonTheAdams the goal amount is an arbitrary number that is used throughout every goal type.  In order for a currency control to make sense, you would need to first make sure the goal type refers to a dollar amount.

Reply:
Hah! Oh yeah. Good catch!

Reply:
Resolved in 5e8217d